packages/at_secondary_server/lib/src/connection/outbound/outbound_client.dart

```diff
@@ -97,15 +98,18 @@ class OutboundClient {
       }
     } on SecondaryNotFoundException catch (e) {
       logger
-          .severe('secondary server not found for $toAtSign: ${e.toString()}');
+          .severe('Secondary server not found for $toAtSign\n${e.toString()}');
```

Comment:
We should avoid breaking log messages over more than one line. Please revert to using colon `:` or pipe `|`

Reply:
reverted all the new-line related changes in [3f36a53](https://github.com/atsign-foundation/at_server/pull/1367/commits/3f36a53d2a44ab80a2ce76e822e77844dba4d5b6)